examples/dataset/face_recognition_11/README.md

```diff
@@ -0,0 +1,87 @@
+# Example Integration: Face Recognition (1:1)
+
+This example integration uses the [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/)
+dataset and Face Recognition (FR) workflow to
+demonstrate how to test and evaluate end-to-end FR (1:1) model pipelines on Kolena.
```

Comment:
We might want to update this now that we are focusing only on the FR evaluation.